src/main/java/com/bookappstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,24 @@
+package com.bookappstore.dto;
+
+import com.bookappstore.validation.Isbn;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
+    @NotBlank
+    private String title;
+    @NotBlank
+    private String author;
+    @NotBlank
+    @Isbn
+    private String isbn;
+    @NotNull
+    @Min(0)
+    private BigDecimal price;
+    private String description;
```

Comment:
You can validate description too

Reply:
not blank + size would be enough, no need for a custom validator I think